src/components/CourseDetail/CourseDetail.js

```diff
@@ -0,0 +1,159 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from "react-router-dom";
+import axios from 'axios';
+
+import ExperienceCard from '../ExperienceCard/ExperienceCard';
+
+/* CourseDetail Component to render the page when a course is selected from a
+    search */
+const CourseDetail = (props) => {
+
+    const location = useLocation();
+    const expObj = location.state.expObj;
+    const imgLink = location.state.imgLink;
+    const api_url = 'http://localhost:8080/es-api/more-like-this/';
+    let cardSection = (
+        <div>
+            Error Loading Related card.
```

Comment:
Probably should have a better error message here indicating an action for the user to take.